.github/workflows/test.yml

```diff
@@ -28,12 +28,42 @@ jobs:
       - name: Test
         run: make test
 
-      - name: Track docs
+      - name: Test if it's a release PR
+        id: check
         run: |
-          if ! [ -z "$(git diff traefik/Chart.yaml | grep version:)" ]; then
-            make docs
-            if ! [ -z "$(git status | grep traefik/VALUES.md)" ]; then
+          set +e
+          changed=$(git diff --name-only -r HEAD^1 HEAD | grep Chart.yaml)
+          if [[ -n $changed ]] ; then
+            echo "release=true" >> "$GITHUB_OUTPUT"
+          fi
+          set -e
+
+      - name: Check docs
+        if: steps.check.outputs.release
+        run: |
+          set +e
+          make docs
+          changed=$(git status | grep VALUES.md)
+          if [[ -n $changed ]]; then
             echo "Traefik values have changed. Run make docs"
+            git diff
             exit 1
-            fi
           fi
+          set -e
+          exit 0
+
+      - name: Create kind cluster
+        if: steps.check.outputs.release
+        uses: helm/kind-action@v1.8.0
+
+      - name: Install LB on Kind
+        if: steps.check.outputs.release
+        run: |
+          kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.13.11/config/manifests/metallb-native.yaml
```

Comment:
We may want to add a renovate config on this.

Reply:
Yes, but probably in an other PR :).